newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Sigsa7_reviews'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '5cbd8b69ee96335a1c9eacca7da172e4ca813663',
```

Comment:
@LittleJennie do you want to add this to your gitignore

Reply:
I think this is ignored.